@tds/core-colours issue

I am getting an issue trying to run npm install command, I have some libraries from @tds, and they have @tds/core-colours as dependency, the issue is that npm is indicating an issue with @tds/colours, so, I am not able to install and to compile.

`> @tds/core-colours@2.2.0 postinstall C:\Users\user\AppData\Roaming\npm\node_modules\@tds\core-colours
> node ./deprecate.js; exit 0;

internal/modules/cjs/loader.js:582
    throw err;
    ^

Error: Cannot find module 'C:\Users\user\AppData\Roaming\npm\node_modules\@tds\core-colours\deprecate.js;'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:580:15)
    at Function.Module._load (internal/modules/cjs/loader.js:506:25)
    at Function.Module.runMain (internal/modules/cjs/loader.js:741:12)
    at startup (internal/bootstrap/node.js:285:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:739:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @tds/core-colours@2.2.0 postinstall: `node ./deprecate.js; exit 0;`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @tds/core-colours@2.2.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\user\AppData\Roaming\npm-cache\_logs\2019-10-08T13_37_35_918Z-debug.log`

--- a/packages/colours/package.json
+++ b/packages/colours/package.json
@@ -10,7 +10,7 @@
     "url": "git+https://github.com/telusdigital/tds.git"
   },
   "scripts": {
-    "postinstall": "node ./deprecate.js; exit 0;"
+    "postinstall": "node deprecate.js"
   },
   "publishConfig": {
     "access": "public"

